sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/communication/Topic.groovy

```diff
@@ -11,7 +11,8 @@ enum Topic {
     PROJECT_REMOVED("A project was removed"),
     NOTIFICATION_FAILURE("Failure notification"),
     NOTIFICATION_SUCCESS("Success notification"),
-    NOTIFICATION_INFO("Information notification")
+    NOTIFICATION_INFO("Information notification"),
+    SAMPLE_RECEIVED_COUNT_UPDATE("The count of received samples was updated")
```

Comment:
The topic does not only add but also remove sample received count. So it addresses an update of the number in general

Reply:
But the question would then be if we want to remove samples from the count if the are not in the status anymore. Which would not make sense in my opinion: if a sample reached a status they will have it, they can only progress.

Reply:
I do not see the point of having a topic for only adding new samples to the status. Then you do not need the count as it will always be 1? You would have to store the current number in your view. By having an update/refresh topic the subscriber can rely on messages reflecting the IS count and not depending on old counts or messages.

Reply:
If a status was completed it does not matter if a sample is currently not in that status anymore, its only about to know that all samples passed